R/download-oni.R

```diff
@@ -1,6 +1,10 @@
 #' @export
 #' @title Download Oceanic Nino Index data
 #' 
+#' @param cache logical option to save and load from cache. If `TRUE`, results will be cached in memory
```

Comment:
What would you think about `use_cache` instead of just `cache`? I think that might communicate intent a little more clearly.

Reply:
Sounds good!